.github/workflows/conda_cpu_nigthly.yaml

```
@@ -52,6 +52,8 @@ jobs:
     - name: Conda-Build@MacOS
       if: startsWith(matrix.os, 'macOS')
       shell: bash -l {0}
+      env:
+         MACOSX_DEPLOYMENT_TARGET: 10.15
```

```suggestion
         MACOSX_DEPLOYMENT_TARGET: 10.13
```

would this still work if we stuck with 10.13? imo we should be as backwards-compatible as possible

---

I don‘t think this is feasible, see #113